starter/script.js

```diff
@@ -1 +1,98 @@
-// put code here!
+firstTimer();
+function firstTimer(){
+  var temp = prompt("Welcome to Washington, D.C.! I hope you're excited to be be here. Are you visiting when the temperature is hot or cold? Enter 'H' for hot or 'C' for cold.");
+  if (temp === "H") {
+    location1();
+  }
+  else if (temp === "C") {
+    location2();
+  }
+}
+//==============================================
+function location1(){
+    var place = prompt("Great! Do you prefer to be inside or outside? Enter 'I' for inside or 'O' for outside.");
+    if (place === "O") {
+      boatOrRUn();
+    }
+    else if (place === "I") {
+      museumOrHouse();
+    }
+    else {
+      alert("Please enter either 'I' or 'O'.");
+      location1();
```

Comment:
I like this recursive way of keeping the prompt going in the case of a bad input. Can you think of other tools at your disposal that could accomplish the same thing?